incipyt/__main__.py

```diff
@@ -122,6 +148,41 @@ def main(folder, verbose, silent, vcs, env, build, license, **kwargs):  # noqa:
     logger.info("All done.")
 
 
+def feed_environ(options, kwargs):
+    project.environ.feed_cli(dict([option.partition("=")[::2] for option in options]))
```

Comment:
The moment the "obvious" solution to 5AM me is `option.partition("=")[::2]` and not `option.split("=", 1)`...